src/components/Select/Select.js

```diff
@@ -78,7 +78,9 @@ const Select = React.forwardRef(
             {children}
           </select>
           {componentsX ? (
-            <ChevronDownGlyph className={`${prefix}--select__arrow`}>
+            <ChevronDownGlyph
+              className={`${prefix}--select__arrow`}
+              aria-label={iconDescription}>
```

Comment:
@emyarod Guessing you actually wanted `aria-label` here, but please speak up if you had removed this deliberately. Thanks!